KinSDK/KinSDK/BulkPayments/BatchPaymentTransaction.swift

```diff
@@ -0,0 +1,20 @@
+//
```

Comment:
KinSDK/KinSDK/**_BulkPayments_**/BatchPaymentTransaction.swift 🥺😞

Reply:
😂
quickly put a folder together to throw everything into. it wont be here in the end once i start organizing the files. and it wont be bulk either 😛